dex/src/main/scala/com/wavesplatform/dex/api/ApiMessage.scala

```diff
@@ -0,0 +1,10 @@
+package com.wavesplatform.dex.api
+
+import io.swagger.annotations.ApiModelProperty
+import play.api.libs.json.{Json, OFormat}
+
+case class ApiMessage(@ApiModelProperty() message: String) extends AnyVal
```

Comment:
Do we really need `@ApiModelProperty()` ? In `OrderV3` we have `amount`, `price` and other fields without such annotation (Does this work?)

Reply:
Empty annotation makes fields not required. We can replace them by `ApiModelProperty(required = false)` but its not so compact

Its correct to make fields of output entities not required, since we actually don't accept them as an input

Meanwhile for the input entites like `Order` or `CancelOrderRequest` (some fields) we make fields required